sos/report/plugins/powerpc.py

```diff
@@ -74,7 +77,10 @@ def setup(self):
                 "serv_config -l",
                 "bootlist -m both -r",
                 "lparstat -i",
-                "ctsnap -xrunrpttr -d %s" % (ctsnap_path)
+                "ctsnap -xrunrpttr -d %s" % (ctsnap_path),
+                "lsdevinfo",
+                "journalctl",
+                "systemctl is-enabled hcn-init.service"
```

Comment:
There is `Plugin.add_service_status()`, which should provide this information while also using the existing policy hooks.

Reply:
sure...will update this with add_service_status() function